build.sbt

```diff
@@ -36,16 +40,21 @@ libraryDependencies ++= Seq(
     "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion,
     "com.typesafe.slick" %% "slick" % SlickVersion,
     "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-    "org.scala-lang.modules" %% "scala-async" % "0.9.5",
+    "org.scala-lang.modules" %% "scala-async" % "0.9.7",
     "org.xerial.snappy" % "snappy-java" % "1.1.1.6",
     "com.github.shyiko" % "mysql-binlog-connector-java" % "0.4.1",
     "org.slf4j" % "slf4j-api" % "1.7.5",
 
     "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % "test",
     "com.h2database" % "h2" % "1.4.189" % "test",
-    "org.scalatest" %% "scalatest" % "2.2.6" % "test",
-    "org.scalacheck" %% "scalacheck" % "1.12.5" % "test"
+    "org.scalactic" %% "scalactic" % "3.1.0",
+    "org.scalatest" %% "scalatest" % "3.1.0" % "test",
+    "org.scalacheck" %% "scalacheck" % "1.14.1" % "test",
+    "org.scalatestplus" %% "scalatestplus-scalacheck" % "3.1.0.0-RC2" % "test"
```

Comment:
What is the purpose of this new library?

Reply:
This was necessitated by the scalatest version bumps. It used to be that many of the property driven methods (i.e. forAll) that Repos uses were located in scalatest, but the library maintainers moved them to the scalatestplus-scalacheck library. 
This change is described in the [3.0.6 release notes](http://www.scalatest.org/release_notes/3.0.6).